Gopkg.toml

```diff
@@ -19,14 +19,17 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+[[override]]
+  source = "https://github.com/fsnotify/fsnotify/archive/v1.4.7.tar.gz"
```

Comment:
Huh? Why is this tar file needed? Can't you just say `version = v1.4.7`?

Reply:
@jodh-intel : Because post `dep ensure`, I got 
```
panic: version queue is empty, should not happen: gopkg.in/fsnotify.v1 

goroutine 1 [running]:
github.com/golang/dep/gps.(*solver).findValidVersion(0xc000086500, 0xc000292000, 0xc0001cadf0, 0x1, 0x1, 0x0, 0xc0001cadf0)
	/home/nitesh/split_patch/src/github.com/golang/dep/gps/solver.go:915 +0x4a0
```

Solution: https://github.com/golang/dep/issues/1799#issuecomment-381026769

Reply:
*Ouch*.

Reply:
This might be "a" fix, but I'm not personally comfortable with doing something like this unless we understand precisely why we cannot vendor the changes "normally".

What do others think?